.github/pull_request_template.md

```diff
@@ -0,0 +1,17 @@
+Please note that we do not accept pull requests for the `master` branch. 
+You will need to raise the PR for the `dev` branch only.
+
+See our [contributing guide](https://github.com/novoic/blabla/blob/master/CONTRIBUTING.md) for details.
+
+Please include the following details in your Pull Request so we are aware of your contribution to BlaBla.
+
+## Desciption
+Please incldue a brief description about the intent behind the Pull Request. For example, you could either be raising an issue, implementing a bugfix or you may be implementing an enhancement in BlaBla.
+
+## Issues that your PR fixes
+Please list the reported issues/bugs the PR fixes with # references. (e.g., #123). Please make sure you have the issue raised before raising the PR.
+
+## Changes in BlaBla as a result of the PR
+Please let us know if you think the PR raises any new issues in BlaBla and how can we handle it. Novoic will do its best to address this if it can be resolved.
+
+Please note that before your Pull Request will be merged, you you must sign a [Contributor License Agreement](https://en.wikipedia.org/wiki/Contributor_License_Agreement) (CLA)
```

Comment:
```suggestion

Please note that before your pull request will be merged, you must sign our [Contributor License Agreement](https://gist.github.com/jackanth/e707f8c0dbe7f1768b4e8c6d28d04170) (CLA). Instructions on how to do this will be automatically commented on the PR once you make it.
```